src/main/resources/reference.conf

```diff
@@ -44,5 +44,20 @@ resourceTypes =
           roleActions = ["launch_notebook"]
         }
       ]
+    },
+    {
+      name = "notebook-cluster"
+      actions = ["status", "connect", "sync", "delete", "read_policies"]
+      ownerRoleName = "owner"
+      roles = [
+        {
+          roleName = "owner"
+          roleActions = ["status", "connect", "sync", "delete", "read_policies"]
+        },
+        {
+          roleName = "projectOwner"
```

Comment:
major nitpick: for formatting consistency with other role names in this config, use `project-owner`

Reply:
actually, leave out project owner for now. I think we will have a different solution.